templates/python/http/func.py

```diff
@@ -1,4 +1,46 @@
 from parliament import Context
+from flask import Request
+
+
+# parse request body, json data or URL query parameters
+def payload_print(req: Request) -> str:
+    if req.method == "POST":
+        if req.is_json:
+            return str(req.json)
+        else:
+            # MultiDict needs some iteration
+            ret = "{"
```

Comment:
Couldn't be this JSON crafting replaced by `json.dumps(req.{form,args})`?

Reply:
flask/werkzeug returns an `ImmutableMultiDict` for both, `form` and `args` - I haven't tried if `json.dumps()` transforms this data structure into the same representation. I went for the JSON crafting to make sure every valid request returns data formatted the exactly same way.

Reply:
And does it returns actual JSON, I see it uses `'` not `"`.

Reply:
It looks like it returns python dict literal.

Reply:
For pretty print only it's OK I guess. But for date sent over write I would prefer JSON.